<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>369</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="tblAccounts">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Username</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Password</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QRadioButton" name="rbDeviceApps">
     <property name="text">
      <string>The applications from this device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pbAddApp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListAdd"/>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pbDeleteApp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListRemove"/>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QRadioButton" name="rbOtherApps">
     <property name="text">
      <string>Other applications</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="cbApp"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pbAddAccount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListAdd"/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pbPasswordVisible">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lePassword"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="leUser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
